docs/root/configuration/operations/overload_manager/overload_manager.rst

```diff
@@ -213,11 +213,23 @@ again 600 seconds, then the minimum timer value would be :math:`10\% \cdot 600s
 Limiting Active Connections
 ---------------------------
 
-Currently, the only supported way to limit the total number of active connections allowed across all
-listeners is via specifying an integer through the runtime key
-``overload.global_downstream_max_connections``. The connection limit is recommended to be less than
+To limit the total number of active downstream connections allowed across all
+listeners configure :ref:`downstream connections monitor <envoy_v3_api_msg_extensions.resource_monitors.downstream_connections.v3.DownstreamConnectionsConfig>` in Overload Manager:
+
+.. code-block:: yaml
+
+   resource_monitors:
+     - name: "envoy.resource_monitors.global_downstream_max_connections"
+       typed_config:
+         "@type": type.googleapis.com/envoy.extensions.resource_monitors.downstream_connections.v3.DownstreamConnectionsConfig
+         max_active_downstream_connections: 1000
+
+One could also set this limit via specifying an integer through the runtime key
+``overload.global_downstream_max_connections``, though this key is deprecated and will be removed in future.
+The connection limit is recommended to be less than
```

Comment:
Might be worth calling out difference that:

One capability that the runtime key has the the existing mechanism doesn't is the ability to get values set during runtime. During deprecation we could see if that is being used by anyone.